src/Mapbender/WmsBundle/Resources/translations/messages.de.yaml

```diff
@@ -162,3 +168,36 @@ mb:
         description: Dimensions-Handler gruppiert und steuert in Der Karte Source Instanzen
     dimensionsets:
       no_dimension: Keine Dimension vorhanden.
+  core:
+    dimensionset:
+      admin:
+        title: Titel
+        group: Gruppe
+        extent: Ausmaß
+    dimensionshandler:
+      admin:
+        tooltip: Tooltip
+        dimensionsets: Dimensionssets
+    wmsloader:
+      admin:
+        defaultformat: Standardformat
+        defaultinfoformat: Standard-Infoformat
+    admin:
+      ? ''
```

Comment:
This yields an error when starting the application:

`An exception has been thrown during the rendering of a template ("The file "/var/www/mapbender/application/mapbender/src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml" does not contain valid YAML: Complex mappings are not supported at line 184 (near "? ''").").`

Same in english version